### PR TITLE
Fix mixed_db_client memory leak.

### PR DESCRIPTION
**Pull request by liuh-80**
Fix mixed_db_client memory leak.

#### Why I did it
Found memory leak in mixed_db_client.

#### How I did it
Add delete code for all sonic-swss-common lib resource.

#### How to verify it
Manually test.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

#### Description for the changelog
Fix mixed_db_client memory leak.

### CODE REVIEW

**Comment by liuh-80** [comment]
All PR blocked by build issue:

github.com/sonic-net/sonic-gnmi/gnmi_server imports
	github.com/Azure/sonic-mgmt-common/translib imports
	github.com/Azure/sonic-mgmt-common/translib/transformer imports
	inet.af/netaddr: missing go.sum entry for module providing package inet.af/netaddr (imported by github.com/Azure/sonic-mgmt-common/translib/transformer); to add:
	go get github.com/Azure/sonic-mgmt-common/translib/transformer@v0.0.0-00010101000000-000000000000

File: sonic_data_client/client_test.go [hunk 1]
```diff
@@ -402,9 +402,11 @@ func ReceiveFromZmq(consumer swsscommon.ZmqConsumerStateTable) (bool) {
 		if receivedData.Size() == 0 {
 			retry++
 			if retry >= 10 {
+				swsscommon.DeleteKeyOpFieldsValuesQueue(receivedData)
```
Comment (ganglyu) [inline sonic_data_client/client_test.go:405]: We don't need to delete before return.
We can use "defer swsscommon.DeleteKeyOpFieldsValuesQueue(receivedData)" after NewKeyOpFieldsValuesQueue.

Comment (liuh-80) [reply sonic_data_client/client_test.go]: Fixed.

**Comment by liuh-80** [comment]
Verified with draft PR: https://github.com/sonic-net/sonic-gnmi/pull/231
No memory leak in ZMQ related code.

**Review by ganglyu — APPROVED** [review]

**Pull request merged**